doc/notes/snmptrapd.md

```diff
@@ -0,0 +1,12 @@
+## Snmptrapd
+
+If you want to enable snmptrapd, you have to run a "sidecar" container (see snmptrapd service in [docker-compose.yml](../../examples/compose/docker-compose.yml) example) or run a simple container like this :
+
+```bash
+docker run -d --name librenms_snmptrapd \
+  --env-file $(pwd)/librenms.env \
+  -e SIDECAR_SNMPTRAPD=1 \
+  -p 162 -p 162/udp \
```

Comment:
Remove `-p 162` .... SNMP is UDP only.

Might be an idea to change to 10161+10162 for DTLS encrypted traps.

Reply:
SNMP isn't udp only. I'm not intrested in DTLS, so feel free to build that ontop of this in your own PR.